meta-phosphor/conf/machine/include/obmc-bsp-common.inc

```diff
@@ -13,7 +13,7 @@ UBOOT_SUFFIX ?= "bin"
 
 MACHINEOVERRIDES =. "openbmc:"
 
-IMAGE_FSTYPES += "overlay"
+MACHINE_FEATURES += "image-overlay"
```

Comment:
I understand the concern.  

I think I originally created the "openpower-pflash" machine feature.  I could make an argument that the "machine" runs the op-build firmware which requires pflash for management.  But I could also be persuaded to use something else entirely.

Agree that overlay isn't really a machine feature.  I think I was just following the first-found pattern.

Should I create arbitrary variable names?  I don't know what the best approach is here.  'overlay' and 'openpower-pflash' do not seem like distro features.  'overlay' could be considered an image feature, but it also needs some component of the machine with it at this point in time because qemu doesn't support the overlay partitioning.  Looking for input.